<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineConfigurationWindow</class>
 <widget class="QWidget" name="MachineConfigurationWindow">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>1022</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Machine Icon Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Machine Configuration</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="attributes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="rowWrapPolicy">
         <enum>QFormLayout::DontWrapRows</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nameEditLine">
          <property name="font">
           <font>
            <bold>false</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="maxLength">
           <number>32</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="coresLabel">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>CPU Cores</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="coresSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="computationPowerLabel">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>CPU Power</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="computationPowerSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> MFLOP/s</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Load Factor</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="loadFactorSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>GPU Cores</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QSpinBox" name="gpuCoresSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <number>1048576</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>GPU Power </string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QSpinBox" name="gpuPower">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string>  MFLOP/s</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>1048576</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>GPU Bandwidth</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QSpinBox" name="gpuBandwidth">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> MByte/s</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1048576</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="ramLabel">
          <property name="text">
           <string>Memory</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QSpinBox" name="ramSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> GB</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1048576</number>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="diskLabel">
          <property name="text">
           <string>Disk</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QSpinBox" name="diskSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> GB</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>102400</number>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="energyIdleLabel">
          <property name="text">
           <string>Energy Idle</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QSpinBox" name="energyIdleSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> Wh</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1048576</number>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Energy Max</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QSpinBox" name="energyMaxSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> Wh</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1048576</number>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="masterLabel">
          <property name="text">
           <string>Master</string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QCheckBox" name="masterCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="22" column="0">
         <widget class="QLabel" name="schedulerLabel">
          <property name="text">
           <string>Scheduler</string>
          </property>
         </widget>
        </item>
        <item row="22" column="1">
         <widget class="QComboBox" name="schedulersComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAtTop</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
          </property>
          <item>
           <property name="text">
            <string>Round Robin</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="23" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Slaves id</string>
          </property>
         </widget>
        </item>
        <item row="23" column="1">
         <widget class="QPlainTextEdit" name="idPlainText"/>
        </item>
        <item row="24" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>ok</string>
          </property>
         </widget>
        </item>
        <item row="20" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Use generated scheduler</string>
          </property>
         </widget>
        </item>
        <item row="20" column="1">
         <widget class="QCheckBox" name="generatedSchedulerCheckbox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="21" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Select scheduler's file</string>
          </property>
         </widget>
        </item>
        <item row="21" column="1">
         <widget class="QPushButton" name="selectGeneratedBtn">
          <property name="text">
           <string>select</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>coresSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>schedulersComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>579</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>computationPowerSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>diskSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ramSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gpuBandwidth</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gpuPower</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gpuCoresSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>loadFactorSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
